LoginPage.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Formik, Form, Field } from 'formik';
+import * as Yup from 'yup';
+import { useHistory } from 'react-router-dom';
+
+const LoginSchema = Yup.object().shape({
+  email: Yup.string().email('Invalid email').required('Required'),
+  password: Yup.string().min(8, 'Too Short!').max(50, 'Too Long!').required('Required'),
+});
+
+const LoginPage = () => {
+  let history = useHistory();
+
+  return (
+    <div className="login-page">
+      <div className="image-section">
+        {/* Your image here */}
+      </div>
+      <div className="form-section">
+        <h1>Login</h1>
+        <p>Enter your email and password to access your account</p>
+        <Formik
+          initialValues={{
+            email: '',
+            password: '',
+          }}
+          validationSchema={LoginSchema}
+          onSubmit={(values) => {
+            // Here you would usually send the values to your server
+            // For this example, we just redirect to the dashboard
+            history.push('/dashboard');
+          }}
+        >
+          {({ errors, touched }) => (
+            <Form>
```

Comment:
Error messages should be handled in a more user-friendly way. Instead of simply displaying the error message, consider using a toast notification or a modal.